general/design-dev-cycle.md

```diff
@@ -0,0 +1,19 @@
+Product/Design development cycle for source{d} application:
+
+- a) Team Sport
```

Comment:
What does `Team Sport` mean in this context?

Reply:
https://github.com/src-d/guide/blob/master/general/design-guide.md#design-is-a-team-sport

Reply:
oh, nice! maybe add a link to that?

Reply:
It makes sense for sure! This cycle was initially proposed on the scope of Code annotation Tool for App & Product teams collaboration , when at that time we were sharing ideas/methods/processes to move forward rock steady. I know that, by this time, we're evaluating more high-level workflows and creating a new form of getting things done. I believe that this cycle is one of many gears in our big machine that will be subject to analysis, nevertheless, I wouldn't say that is set in stone until proper Management Assessment.

Reply:
Yes, I agree with @campoy it would be nice to have a link to [Design is a team sport](https://github.com/src-d/guide/blob/master/general/design-guide.md#design-is-a-team-sport)